modules/gdscript/editor/script_templates/CharacterBody2D/basic_movement.gd

```diff
@@ -3,8 +3,9 @@
 extends _BASE_
 
 
-const SPEED = 300.0
-const JUMP_VELOCITY = -400.0
+const SPEED : float = 300.0
```

Comment:
I think we should follow the [Writing guidelines](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#dynamic-vs-static-typing) and omit the types. (Even though I am a static types enjoyer.)

A nice improvement to the template system (separate from this PR) would be to offer a variant with type hints depending on the user's settings.

Reply:
I was thinking that with round numbers like this, it makes more sense to secure it with the explicit float rather than depending on `:=` in case this ends up being changed to a round integer constant `300` which may cause weird bugs down the line

I can also drop the commit and keep the current `=`

Reply:
This will be preprocessed here:

https://github.com/godotengine/godot/blob/be56cab58c056c074d1e02cd0b38641204e39f41/modules/gdscript/gdscript_editor.cpp#L78-L96

Reply:
Commit removed